src/screens/Voucher/VoucherIndexScreen.tsx

```diff
@@ -118,7 +126,7 @@ export const VoucherIndexScreen = ({ navigation, route }: StackScreenProps<any>)
     return <LoadingSpinner loading />;
   }
 
-  const count = listItems?.length;
+  const count = _filter(listItems, ({ categories }) => categories.length > 0).length;
```

Comment:
what about `listItems.filter({ categories } => !!categories.length).length`? i do not think that we need lodash here and on the other side we often not check length with > 0 but with boolean !!

Reply:
There are also two different usages in the code. Both do the same thing. loadsh has no extra advantage. I can change the code. 

Yes you are right, I will update the category control.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/7d0a8ceffce23adcff5deea4e69324d54058c226